en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,59 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
+When it comes to display the value, to list the entities for instance, there is no easy way to render
```

Comment:
which entities are you talking about ? You really should given a clear and exhaustive example of the issue.